javascript/javascript2/week1/main.js

```diff
@@ -0,0 +1,29 @@
+console.log("Script loaded");
+
+const products = getAvailableProducts();
+
+function renderProducts(products) {
+    const productsList = document.getElementById("list");
+    for (let i = 0; i < products.length; i++) {
+        // Create the list item:
+        const item = document.createElement("li");
+
+        // Formatting product details to print:
+        item.innerHTML =
+            "<h2>" + products[i].name +
+            "<br> " + "</h2>" +
+            " price:" +
+            products[i].price +
+            " <br> " +
+            " rating:" +
+            products[i].rating;
+
+        // Add the product to productsList:
+        productsList.appendChild(item);
+    }
+
+
+    return productsList;
+}
+
+renderProducts(products); // This should create the ul and the li's with the individual products details
```

Comment:
This is ok, but keep in mind that the <br> tag is now a little old fashioned👴. The **box model** has taken over for it (in the css part, no need to worry about this now).